utils/validation.py

```diff
@@ -98,27 +98,27 @@ def validate_tick(tick, previous_tick=None, possible_previous_ticks=None,
 
     if not validate_schema(tick_copy, 'tick_schema.json'):
         logger.debug("Tick failed schema validation")
-        return False
+        return False, False
```

Comment:
See comments in flask_api for why you can't use this construction (return only 1 bool pls)